include/envoy/network/transport_socket.h

```diff
@@ -33,6 +33,11 @@ struct IoResult {
    * can only be true for read operations.
    */
   bool end_stream_read_;
+
+  /**
+   * True if the I/O call blocked.
+   */
+  bool didBlock_;
```

Comment:
I think would_block_ describes this more accurately. All of the IO calls should be non-blocking and will return EWOULDBLOCK if the operation were to block. So no blocking should actually occur

Reply:
Good point, will change

Reply:
fixed in f36dad1d76eb51ef5563089ab14de1ff9eae8800